src/app/core/common-components/entity-fields-menu/entity-fields-menu.component.html

```diff
@@ -1,27 +1,9 @@
-<button
-  mat-icon-button
-  [matMenuTriggerFor]="list"
-  matTooltip="Add or remove visible fields"
-  i18n-matTooltip
-  class="square-button"
->
-  <fa-icon [icon]="icon" transform="shrink-4"></fa-icon>
-</button>
-
-<mat-menu #list>
-  <button
-    mat-menu-item
-    *ngFor="let field of _availableFields"
-    (click)="toggleFieldSelection(field)"
-  >
-    <fa-icon
-      class="standard-icon-with-text"
-      [icon]="activeFields.includes(field.id) ? 'toggle-on' : 'toggle-off'"
-    ></fa-icon>
-
-    <app-entity-field-label
-      [field]="field"
-      [entityType]="entityType"
-    ></app-entity-field-label>
-  </button>
-</mat-menu>
+<mat-form-field>
+  <app-basic-autocomplete
```

Comment:
isn't this basically the `EntityFieldSelectComponent` now?

Reply:
Yes, it's the same. I'm still using EntityFieldsMenuComponent for now because we might want to customize it in the future.
But what do you think, should we use EntityFieldSelectComponent instead? If so, we can remove this.

Reply:
Having a basis to customize again is a fair point. How about using <app-entity-field-select> within the template of this component for now?